node-app/templates/webhook.yaml

```diff
@@ -0,0 +1,42 @@
+{{ $ca := genCA "svc-cat-ca" 3650 }}
+{{ $webhookCert := genSignedCert "topokube-node-app.topokube.svc.cluster.local" nil (list "topokube-node-app.topokube.svc" "topokube-node-app.topokube") 365 $ca }}
```

Comment:
I realized I was using this incorrectly. useful docs: https://masterminds.github.io/sprig/crypto.html#gensignedcert